src/en/guide/basics/headers.md

```diff
@@ -25,23 +31,57 @@ $ curl localhost:8000 \
     -H "Authorization: Token ABCDEF12345679"
 ABCDEF12345679
 ```
+
 ```bash
 $ curl localhost:8000 \
     -H "Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"
 eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
 ```
+
 :---
+
 #### Proxy headers
+
 Sanic has special handling for proxy headers. See the [proxy headers](/guide/advanced/proxy-headers.md) section for more details.
 
 ---:1
+
+#### Other headers
+
+Sanic support you operate headers, Header keys are converted to *lowercase* when parsed. Capitalization is not considered for headers.
+You can operate it like dict.
+
+:--:1
+
+```python
+@app.route("/")
+async def handler(request):
+    print(request.headers["user-agent"])
+    print(request.headers.items())
+    return json(dict(request.headers))
+```
+
+```bash
+$ curl localhost:8000 
+```
+
+```bash
+$ curl localhost:8000 
+{"host":"localhost:8000","user-agent":"curl/7.64.1","accept":"*/*"}
+```
+
+:---
+
 #### Request ID
```

Comment:
Let's also include here the same `FYI` text we use with query arguments, form data, and request files:

> FYI
>
> 💡 The request.args object is one of a few types that is a dictionary with each value being a list. This is because HTTP allows a single key to be reused to send multiple values.
>
> Most of the time you will want to use the .get() method can be used to access the first element and not a list. If you do want a list of all items, you can user .getlist().

Reply:
professional! let me re-edit it